src/main/scala/rocket/CSR.scala

```diff
@@ -297,6 +297,7 @@ class CSRFile(
   val reset_mstatus = Wire(init=new MStatus().fromBits(0))
   reset_mstatus.mpp := PRV.M
   reset_mstatus.prv := PRV.M
+  reset_mstatus.xs := if (usingRoCC) UInt(3) else UInt(0)
```

Comment:
I think this won't compile unless you add parens around the if/else.

Reply:
:man_facepalming: